package.json

```diff
@@ -29,7 +29,7 @@
 		"lint": "npm run lintJs && npm run lintStyle",
 		"storybookBuild": "build-storybook -c .storybook -o docs/storybook",
 		"storybook": "start-storybook -c .storybook",
-		"sassdocBuild": "node ./node_modules/sassdoc/bin/sassdoc styles/scss --dest docs/sassdocs",
+		"sassdocBuild": "node ./node_modules/sassdoc/bin/sassdoc styles/scss --dest docs/sassdocs && cp ./styles/sassdocs/main.css ./docs/sassdocs/assets/css/",
```

Comment:
This could be an issue on Windows, because of path specification. We should test this.